deno/payloads/v10/gamesdk.ts

```diff
@@ -0,0 +1,308 @@
+/**
+ * Types extracted from https://discord.com/developers/docs/game-sdk/sdk-starter-guide
+ */
+
+import type { Snowflake } from '../../globals.ts';
+import type { LocalizationMap } from '../common.ts';
+
+export interface APIAchievement {
+	/**
+	 * Unique id of the application
+	 */
+	application_id: Snowflake;
+	/**
+	 * Name of this achievement
+	 */
+	name: string;
+	/**
+	 * Localization dictionary for the `name` field
+	 */
+	name_localizations: LocalizationMap | null;
+	/**
+	 * The description of this achievement
+	 */
+	description: string;
+	/**
+	 * Localization dictionary for the `description` field
+	 */
+	description_localizations: keyof LocalizationMap | null;
```

Comment:
```suggestion
	description_localizations: LocalizationMap | null;
```

Reply:
I actually mixed the things lol
it should be for `APISKU#locales` instead, thanks